release.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+git commit -am "Released version $1"
+git tag -m "Released version $1" $1
+git push origin $1
+
+python setup.py sdist bdist_wheel
```

Comment:
Instead of building two different wheels, if your code is python 2/3 compatible, you should add a `setup.cfg` file and define the project as a universal wheel. Then call bdist_wheel just the one time. See: https://packaging.python.org/tutorials/distributing-packages/#universal-wheels